plugins/in_mqtt/mqtt_prot.c

```diff
@@ -144,15 +144,19 @@ static int mqtt_data_append(char *topic, size_t topic_len,
         return -1;
     }
 
+
+printf("PAYLOAD : ||||||%.*s|||||\n\n", msg_len, msg);
```

Comment:
:)

Reply:
Dang, sorry, I'll push a fix ASAP, thanks for reviewing the code.